aea/manager/manager.py

```diff
@@ -249,20 +272,23 @@ def start_manager(
 
         :param local: whether or not to fetch from local registry.
         :param remote: whether or not to fetch from remote registry.
+
         :return: the MultiAgentManager instance.
         """
         if self._is_running:
-            return self
```

Comment:
That breaks compatibility. Why are we doing this? We could store this info in a property so user can request it.